portfolio/src/main/webapp/index2.html

```diff
@@ -0,0 +1,84 @@
+            </ul>
+        </div>
+        </br>
+        </br> 
+        <div id="div2">
+            <a id="Destiny">
+                <h2>Destiny</h2>
+            </a>
+            <img src="/images/Destiny.jpg", width="900px">
+            </br>
+            <p>Destiny, as a franchise, has become one of my favorite games to play
+                over the nearly six years that the franchise has been around. With this,
+                the action-FPS MMO-lite science fantasy game proves to draw me in with its
+                captivating lore and theorizing potential. While the current content for the 
+                game Destiny 2 has been lacking overall... I will continue to play the major 
+                franchise that I have come to love.
+            </p>
+            </br>
+            <p>My Guardian Class?: Warlock</p>
+        </div>
+        </br>
+        </br> 
+        <div id="div2">
+            <a id="Smite">
+                <h2>Smite</h2>
+            </a>
+            <img src="/images/Smite-Kukulkan.png">
+            </br>
+            <p>Smite is one of the games that I enjoy to play every so often. Whether it be
+                when I'm bored or when I am not playing Destiny, Smite is usually a mostly
+                enjoyable experience. Smite is a MOBA played in the third-person perspective
+                rather than the usual bird's-eye view. It plays pactically like your normal 
+                MOBA like League of Legends or DOTA 2 with very similar mechanics. All of the 
+                characters you play as are gods and goddesses of pantheons long gone...
+                (or various monsters and literary gods). Despite the frustration with balancing,
+                the game proves to be a fun game to sit down and play from time to time.
+            </p>
+            </br>
+            <p>My main in Smite?: Kukulkan</p>
+        </div>
+        </br>
+        </br> 
+        <div id="div2">
+            <a id="Spyro:_Reignited_Trilogy">
+                <h2>Spyro: Reignited Trilogy</h2>
+            </a>
+            <img src="/images/Spyro.jpg", width="900px">
+            </br>
+            <p>Spyro is one of those franchises I've been interested in without playing the main games 
+                of the franchise. Maybe just because of my love for dragons! Having the original
+                three games remastered for modern consoles was an awesome thing to see and immediately 
+                garnered a preorder from me. Being able to finally experience the original trilogy of 
+                games in an updated fashion was an amazing experience. I've always had a fondness
+                of older gamees, and getting to play a remake of of these games was simply...
+                awesome.
+            </P>
+            </br>
+            <p>Favorite Spyro Game?: Spyro 2</p>
+        </div>  
+        </br>
+        </br>  
+        <div id="link_row">
```

Comment:
It looks like this opening <div> doesn't have a corresponding </div> anywhere

Reply:
Did you just inject HTML into GitHub's comment view? I'm assuming you wrote `<div>` in your comment.

<script>alert("pls don't show up")</script>

Reply:
Okay, glad that didn't fire an alert (sorry this has nothing to do with the CL, I just found this amusing).